### PR TITLE
Added viewing of posts dashboard/profile

### PR DESCRIPTION
**Pull request by Hakdoff**
OVERVIEW
Viewing of all posts in dashboard.
Viewing of own posts in profile.

SNIPPETS
image

### CODE REVIEW

File: src/app/Models/Post.php [hunk 1]
```diff
@@ -10,4 +10,9 @@ class Post extends Model
     use HasFactory;
     
     protected $fillable = ['content', 'user_id'];
+    
+    public function user()
```
Comment (mikco-hipe) [inline src/app/Models/Post.php]: add docbloc

Comment (Hakdoff) [reply src/app/Models/Post.php]: Added docbloc.

File: src/app/Models/Post.php [hunk 2]
```diff
@@ -10,4 +10,12 @@ class Post extends Model
     use HasFactory;
     
     protected $fillable = ['content', 'user_id'];
+    
+    /**
+     * Post belongs to user
+     */
+    public function user()
```
Comment (mikco-hipe) [inline src/app/Models/Post.php]:
```suggestion
    public function user(): BelongsTo

**Review by sunkenquest — APPROVED** [review]
LGTM 💯

**Pull request merged**